cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -333,6 +347,14 @@ def test_import_report_rest(appliance):
     }
     response, = appliance.rest_api.collections.reports.action.execute_action("import", data)
     assert_response(appliance)
+
+    @request.addfinalizer
+    def _finalize():
+        report = appliance.collections.reports.instantiate(
```

Comment:
Q: it is not possible to remove `report` with `rest` as test pointing rest?

Reply:
It's not possible, we can only `run` and `schedule` a report entity via REST and `import` a report with ReportCollection via REST..